Documentation/ceph-configuration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuration
+weight: 3700
+indent: true
+---
+
+# Configuration
+
+For most any Ceph cluster, the user will want to -- and may need to -- change some Ceph
```

Comment:
`--` can be filled in?

Reply:
I don't understand. Does this render poorly?